### PR TITLE
feat(blog): Add keymap-editor community spotlight post

### PR DESCRIPTION
**Pull request by caksoylar**
This is the first post in (hopefully) a series of blog posts highlighting the useful projects in the ZMK ecosystem. @nickcoutsos penned the first one introducing his Keymap Editor project.

Note also the commit that removes the post limit on the sidebar showing "Recent Posts". I thought it was too difficult to get to older blog posts (I think you need to go to blog root and scroll down, basically?) and I don't see a downside to removing the limit. Let me know if you feel otherwise.

### CODE REVIEW

**Review by petejohanson — APPROVED** [review]
Love it! Thanks for putting this together!

**Comment by petejohanson** [comment]
@caksoylar Feel free to merge. May want to tweak the blog file date to whenever you merge this so it's accurate.

File: docs/blog/2023-11-07-keymap-editor.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+alt="Shows a screenshot of the Keymap Editor application featuring a graphical layout of the Corne Keyboard with a keymap loaded from the nickcoutsos/keymap-editor-demo-crkbd GitHub repository."
+sources={{
+    light: require('./assets/2023-11-07-keymap-editor/editor-screenshot-light.png').default,
+    dark: require('./assets/2023-11-07-keymap-editor/editor-screenshot-dark.png').default,
+  }}
+/>
+
+This blog post is the first in a series of posts where we highlight projects within the ZMK ecosystem that we think are cool and that the users might benefit from knowing about them. We are starting the series with a big one, [Keymap Editor] by [Nick Coutsos](https://github.com/nickcoutsos)!
+
+In the rest of the post we leave it to Nick himself to introduce the project, detail his goals and motivation in developing such a tool, and talks about the future of the project. Stay tuned for future installments in the series!
```
Comment (caksoylar) [inline docs/blog/2023-11-07-keymap-editor.md:22]:
```suggestion
In the rest of the post we leave it to Nick himself to introduce the project, detail his goals and motivation in developing such a tool, and talk about the future of the project. Stay tuned for future installments in the series!
```

File: docs/blog/2023-11-07-keymap-editor.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+_[Keymap Editor]_ is a web based graphical editor for ZMK keymaps. It provides a visual way to manage the contents of your keymap and if nothing else offers two critical features:
+
+1. Automatic formatting of the keymap file, so that bindings arrays remain readable
+2. Searchable behaviors, keycodes, commands, etc, so you won't have to remember if it's `LCTL` or `LCTRL` (I just had to double check myself and I guessed wrong, apparently)
+
+## What can Keymap Editor do?
+
+- Render [devicetree keymaps](/docs/features/keymaps) using pre-defined, auto-generated, or side-loadable keyboard layouts
+- Integrate with a GitHub repo to streamline firmware builds, or FileSystem/Clipboard if you'd still rather build locally
+- Edit [combos](/docs/features/combos), [behaviors](/docs/behaviors/key-press), [macros](/docs/behaviors/macros), and [conditional layers](/docs/features/conditional-layers), [rotary encoder bindings](/docs/behaviors/sensor-rotate)
```
Comment (caksoylar) [inline docs/blog/2023-11-07-keymap-editor.md:35]:
```suggestion
- Edit [combos](/docs/features/combos), [behaviors](/docs/behaviors/key-press), [macros](/docs/behaviors/macros), [conditional layers](/docs/features/conditional-layers) and [rotary encoder bindings](/docs/behaviors/sensor-rotate)
```

**Comment by caksoylar** [comment]
Thanks @petejohanson! I'll rename and merge later tonight (along with fixing a couple tiny issues I just noticed), at which point it will probably be Nov 9 UTC.

**Pull request merged**